### PR TITLE
Error deleting a group

### PR DESCRIPTION
**Pull request by mrvisser**
The following error shows up in the console:

```
Uncaught Error: Syntax error, unrecognized expression: <!-- MODAL -->
<div id="deleteresource-modal" class="modal fade" tabindex="-1" aria-labelledby="deleteresource-modal-title" aria-hidden="true"><!-- --></div>

<div id="deleteresource-template"><!--
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                    <span aria-hidden="true">&#215;</span>
                </button>
                <h3 id="deleteresource-modal-title">
                    {if contextProfile.resourceType === 'discussion'}
                        Delete discussion
                    {elseif contextProfile.resourceType === 'group'}
                        Delete group
                    {else}
                        {if contextProfile.resourceSubType === 'collabdoc'}
                            Delete document
                        {elseif contextProfile.resourceSubType === 'file'}
                            Delete file
                        {elseif contextProfile.resourceSubType === 'link'}
                            Delete link
                        {/if}
                    {/if}
                </h3>
            </div>
            <div class="modal-body">
                <h4>
                    {if contextProfile.resourceType === 'discussion'}
                        Are you sure you want to delete this discussion?
                    {elseif contextProfile.resourceType === 'group'}
                        Are you sure you want to delete this group?
                    {else}
                        {if contextProfile.resourceSubType === 'collabdoc'}
                            Are you sure you want to delete this document?
                        {elseif contextProfile.resourceSubType === 'file'}
                            Are you sure you want to delete this file?
                        {elseif contextProfile.resourceSubType === 'link'}
                            Are you sure you want to delete this link?
                        {/if}
                    {/if}
                </h4>
                <div class="alert alert-danger text-center">
                    <strong>Caution:</strong> this action cannot be undone!
                </div>
            </div>
            <div class="modal-footer">
                <button type="button" class="btn btn-link" data-dismiss="modal">Cancel</button>
                <button type="button" id="deleteresource-delete" class="btn btn-danger">
                    {if contextProfile.resourceType === 'discussion'}
                        Delete discussion
                    {elseif contextProfile.resourceType === 'group'}
                        Delete group
                    {else}
                        {if contextProfile.resourceSubType === 'collabdoc'}
                            Delete document
                        {elseif contextProfile.resourceSubType === 'file'}
                            Delete file
                        {elseif contextProfile.resourceSubType === 'link'}
                            Delete link
                        {/if}
                    {/if}
                </button>
            </div>
        </div>
    </div>
--></div>

<div id="deleteresource-notification-title-template"><!--
    {if err}
        {if contextProfile.resourceType === 'discussion'}
            Discussion not deleted.
        {elseif contextProfile.resourceType === 'group'}
            Group not deleted.
        {else}
            {if contextProfile.resourceSubType === 'collabdoc'}
                Document not deleted.
            {elseif contextProfile.resourceSubType === 'file'}
                File not deleted.
            {elseif contextProfile.resourceSubType === 'link'}
                Link not deleted.
            {/if}
        {/if}
    {else}
        {if contextProfile.resourceType === 'discussion'}
            Discussion deleted.
        {elseif contextProfile.resourceType === 'group'}
            Group deleted.
        {else}
            {if contextProfile.resourceSubType === 'collabdoc'}
                Document deleted.
            {elseif contextProfile.resourceSubType === 'file'}
                File deleted.
            {elseif contextProfile.resourceSubType === 'link'}
                Link deleted.
            {/if}
        {/if}
    {/if}
--></div>

<div id="deleteresource-notification-body-template"><!--
    {if err}
        {if contextProfile.resourceType === 'discussion'}
            The discussion could not be deleted.
        {elseif contextProfile.resourceType === 'group'}
            The group could not be deleted.
        {else}
            {if contextProfile.resourceSubType === 'collabdoc'}
                The document could not be deleted.
            {elseif contextProfile.resourceSubType === 'file'}
                The file could not be deleted.
            {elseif contextProfile.resourceSubType === 'link'}
                The link could not be deleted.
            {/if}
        {/if}
    {else}
        {if contextProfile.resourceType === 'discussion'}
            The discussion has been successfully deleted.
        {elseif contextProfile.resourceType === 'group'}
            The group has been successfully deleted.
        {elseif contextProfile.resourceType === 'content'}
            {if contextProfile.resourceSubType === 'collabdoc'}
                The document has been successfully deleted.
            {elseif contextProfile.resourceSubType === 'file'}
                The file has been successfully deleted.
            {elseif contextProfile.resourceSubType === 'link'}
                The link has been successfully deleted.
            {/if}
        {/if}
    {/if}
--></div>

<!-- JAVASCRIPT -->
<script type="text/javascript" src="https://qa0-researchresearch-researchresearch.netdna-ssl.com/node_modules/oae-core/deleteresource/js/deleteresource.1398ef31.js"></script>
```

### CODE REVIEW

**Comment by mrvisser** [comment]
Assigned to @nicolaasmatthijs for review

**Comment by nicolaasmatthijs** [comment]
@mrvisser: Can you also remove the empty newlines at the beginning of `deletecomment.html`, `deletefolder.html` and `deleteresource.html`?

File: shared/oae/api/oae.api.widget.js [hunk 1]
```diff
@@ -393,7 +393,7 @@ define(['exports', 'jquery', 'underscore', 'oae.api.config', 'oae.api.i18n', 'oa
             // We transform the translated HTML into a jQuery object. However, as this will actually load all of the
             // images in the widget HTML straight away, and the images will still have relative URLs that need conversion,
             // we replace all `img` tags to temporary `imgtmp` tags
-            widgetHTML = widgetHTML.replace(/<img/ig, '<imgtmp');
+            widgetHTML = widgetHTML.replace(/<img/ig, '<imgtmp').trim();
```
Comment (nicolaasmatthijs) [inline shared/oae/api/oae.api.widget.js]: Would you mind doing the `.trim` first and then doing the `replace`. It makes it slightly easier to understand what's going on

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @mrvisser for follow-up

**Comment by mrvisser** [comment]
Made changes. Back to @nicolaasmatthijs

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged

**Pull request merged**